game-core/src/test/java/games/strategy/engine/lobby/client/ui/LobbyGameTableModelTest.java

```diff
@@ -4,96 +4,111 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.nullValue;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.mockito.Mockito.when;
 
 import com.google.common.util.concurrent.Runnables;
-import games.strategy.engine.message.IChannelMessenger;
-import games.strategy.engine.message.IRemoteMessenger;
-import games.strategy.engine.message.MessageContext;
-import games.strategy.net.IMessenger;
-import games.strategy.net.INode;
-import games.strategy.net.Messengers;
+import games.strategy.net.Node;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
 import java.time.Instant;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.UUID;
+import java.util.ArrayList;
+import java.util.List;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.triplea.http.client.lobby.game.listing.GameListingClient;
+import org.triplea.http.client.lobby.game.listing.LobbyGameListing;
 import org.triplea.java.Interruptibles;
 import org.triplea.lobby.common.GameDescription;
-import org.triplea.lobby.common.ILobbyGameController;
 import org.triplea.swing.SwingAction;
 import org.triplea.util.Tuple;
 
 final class LobbyGameTableModelTest {
+
+  private static final String id0 = "id0";
+  private static final String id1 = "id1";
+  private static final GameDescription gameDescription0;
+  private static final GameDescription gameDescription1;
+
+  static {
+    try {
+      gameDescription0 =
+          GameDescription.builder()
+              .hostedBy(new Node("node", InetAddress.getLocalHost(), 10))
+              .startDateTime(Instant.now())
+              .status(GameDescription.GameStatus.LAUNCHING)
+              .build();
+
+      gameDescription1 =
+          GameDescription.builder()
+              .hostedBy(new Node("another node", InetAddress.getLocalHost(), 10))
+              .startDateTime(Instant.now())
+              .status(GameDescription.GameStatus.IN_PROGRESS)
+              .build();
+    } catch (final UnknownHostException e) {
+      throw new RuntimeException(e);
+    }
+  }
```

Comment:
I don't know if it is necessarily more idiomatic, fundamentally all we need is a static block. IMO better to stick to the language primitives and the minimum required to get the job done. A `@BeforeAll` has advantages if there are mock objects involved, in this case we just need something for the `catch` block (ideally the `Node` API could have handled this for us, but it does not).

Reply:
Yeah, I'm just not sure what would happen in case an actual exception occurs.
Using a separate method would make it clear that a test case failed, while an exception during static initialization would probably result in a confusing stacktrace

Reply:
> Using a separate method would make it clear that a test case failed,

Not quite, as all test cases can be marked as failed, going to the stack trace shows it was a beforeAll method. On the other hand, an initializer block tends to have the other test cases be aborted. Either way the stack trace would be the thing to lead someone to the RC. To another extent, we expect 'getLocalHost' to always provide a valid hostname.